impeller/scene/geometry.h

```diff
@@ -8,17 +8,27 @@
 
 #include "impeller/geometry/vector.h"
 #include "impeller/renderer/allocator.h"
+#include "impeller/renderer/device_buffer.h"
 #include "impeller/renderer/vertex_buffer.h"
+#include "impeller/scene/importer/scene_flatbuffers.h"
 
 namespace impeller {
 namespace scene {
 
 class CuboidGeometry;
+class VertexBufferGeometry;
 
 class Geometry {
```

Comment:
I know this is unrelated to this patch so we can fix this later, but, `Geometry` should have a virtual destructor.